### PR TITLE
release notes for 4.0.0

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 1]
```diff
@@ -13,15 +13,26 @@ These pages cover the 4.x versions of the Couchbase PHP SDK.
...
-API documentation: https://docs.couchbase.com/sdk-api/couchbase-php-client-4.0.0/
+API documentation: https://docs.couchbase.com/sdk-api/couchbase-php-client-4.0.0
 
-* https://issues.couchbase.com/browse/PCBC-123[PCBC-123]: 
-Sample ticket XXXXXX
+|===
+|Platform|
+|Source          |https://packages.couchbase.com/clients/php/couchbase-4.0.0.tgz[couchbase-4.0.0.tgz]
+|Checksum        |https://packages.couchbase.com/clients/php/couchbase-4.0.0.sha256sum[couchbase-4.0.0.sha256sum]
+|===
```
Comment (maria-robobug) [inline modules/project-docs/pages/sdk-release-notes.adoc:28]: @avsej, thanks for putting this up, do we not have a list of tickets (known issues, or fixed issues etc?)

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 2]
```diff
@@ -13,15 +13,26 @@ These pages cover the 4.x versions of the Couchbase PHP SDK.
...
+multi-document distributed ACID transactions, and bringing a number of improvements related to internal connection
+behavior.
 
-API documentation: https://docs.couchbase.com/sdk-api/couchbase-php-client-4.0.0/
+API documentation: https://docs.couchbase.com/sdk-api/couchbase-php-client-4.0.0
 
-* https://issues.couchbase.com/browse/PCBC-123[PCBC-123]: 
-Sample ticket XXXXXX
+|===
+|Platform|
```
Comment (maria-robobug) [inline modules/project-docs/pages/sdk-release-notes.adoc]: @avsej what Platform did we actually intend to put here?

Comment (avsej) [reply modules/project-docs/pages/sdk-release-notes.adoc]: in previous version of the release notes, this column also contained combinations of Windows platforms (like PHP version, architecture and thread safety variant). For this, 4.0.0 release we don't publish binaries for windows, so this column does not represent any "platform". Feel free to remove this table header completely.

**Comment by maria-robobug** [comment]
I'm going to merge this so we have some release notes, but I still don't have a list of tickets to include in it, but guess will have to do for now, so we can publish...

**Review by maria-robobug — APPROVED** [review]

**Pull request merged**